description: don't regenerate Makefile.am files
author: Michael Gilbert <mgilbert@debian.org>

Index: isc-dhcp/configure.ac
===================================================================
--- isc-dhcp.orig/configure.ac
+++ isc-dhcp/configure.ac
@@ -968,19 +968,15 @@ AC_CONFIG_FILES([
   Makefile
   client/Makefile
   client/tests/Makefile
-  common/Makefile.am
   common/Makefile
   common/tests/Makefile
-  dhcpctl/Makefile.am
   dhcpctl/Makefile
   includes/Makefile
   keama/Makefile
-  omapip/Makefile.am
   omapip/Makefile
   relay/Makefile
   relay/tests/Makefile
   server/Makefile
-  tests/Makefile.am
   tests/Makefile
   tests/unittest.sh
   server/tests/Makefile
