--- isc-dhcp-4.4.3-P1.orig/bind/Makefile.in
+++ isc-dhcp-4.4.3-P1/bind/Makefile.in
@@ -21,14 +21,15 @@
 prefix = @prefix@
 exec_prefix = @exec_prefix@
 
-bindconfig = --without-openssl --without-libxml2 --without-libjson \
+bindconfig = --build=@build_alias@ --host=@host_alias@ CC=@CC@ \
+	--without-openssl --without-libxml2 --without-libjson \
 	--without-gssapi --disable-threads --without-lmdb \
 	--includedir=@includedir@ --libdir=@libdir@  --without-python\
 	@BINDLT@ @BINDIOMUX@ @BINDCONFIG@ --enable-full-report
 
 @BIND_ATF_FALSE@cleandirs = ./lib ./include
 @BIND_ATF_TRUE@cleandirs = ./lib ./include ./atf
-cleanfiles = ./configure.log ./build.log ./install.log
+cleanfiles = /configure.log ./build.log ./install.log
 
 bindlibs = isc dns isccfg irs
 installdirs = includedir=${binddir}/include libdir=${binddir}/lib
